packages/cozy-harvest-lib/src/models/ConnectionFlow.js

```diff
@@ -497,6 +497,26 @@ export class ConnectionFlow {
     )
 
     this.job = await launchTrigger(this.client, this.trigger)
+
+    if (this.konnector.client_side) {
+      if (window.ClientConnectorLauncher === 'react-native') {
+        logger.info('Found client connector react-native launcher')
+        window.ReactNativeWebView.postMessage(
+          JSON.stringify({
+            message: 'startLauncher',
+            value: {
+              connector: this.konnector,
+              account: this.account,
+              trigger: this.trigger,
+              job: this.job
+            }
+          })
+        )
```

Comment:
We should create a `Bridge` or a `CommunicationChannel` between Harvest and the launcher. 

At first we only need to send this `startLauncher` command to a RN environnement, but we can imagine having other channels. 

Also, for now we rely only on realtime socket, but we can easily miss a message on the client side. We would have to listen to event coming from the launcher 

Something similar to CozyRealtime can be imagined: 
```
Bridge.send({
   message: 'startLauncher',
            value: {
              connector: this.konnector,
              account: this.account,
              trigger: this.trigger,
              job: this.job
            }
})

Bridge.subscribe('...')
```

Is it planned in the future?

Reply:
Yes